src/python/pants/backend/python/goals/setup_py.py

```diff
@@ -682,16 +685,28 @@ async def generate_chroot(request: SetupPyChrootRequest) -> SetupPyChroot:
             "install_requires": (*requirements, *setup_kwargs.get("install_requires", [])),
         }
     )
+
+    # Add `.with_binries()` to the entry points.
```

Comment:
```suggestion
    # Add `.with_binaries()` to the entry points.
```

Reply:
Also I don't understand what this comment is saying. `with_binaries()` is in a BUILD file, we're not adding it, whatever that means. Maybe reword?